src/tcpedit/edit_packet.c

```diff
@@ -1202,3 +1202,61 @@ is_multicast_ipv6(tcpedit_t *tcpedit, struct tcpr_in6_addr *addr)
     return 0;
 }
 
+
+/**
+ * based on  which is generated via the range_[src/dst] and range.
```

Comment:
@dsugisawa-mixi need a better description. Also, would it make sense to add IPv6 support?